solutions/gold/cf-1666F.mdx

```diff
@@ -119,8 +119,8 @@ void solve() {
 				if (j + k > n / 2) { continue; }
 				dp[size + b[i].second][j + k] =
 				    (dp[size + b[i].second][j + k] +
-				     choose(positions, b[i].second - k, MOD) *
-				     dp[size][j] % MOD) %
+				     choose(positions, b[i].second - k, MOD) * dp[size][j] %
+				         MOD) %
```

Comment:
this is the WORST way they could've formatted this

Reply:
I can't fix it because whenever I do the pre-commit bot just changes it back.

Reply:
@bqi343 any idea why it formats like this?

Reply:
if you don't want it to look like this then split the statement into multiple statements